src/systemtask/SystemTask.cpp

```diff
@@ -2,7 +2,7 @@
 #include <hal/nrf_rtc.h>
 #include <libraries/gpiote/app_gpiote.h>
 #include <libraries/log/nrf_log.h>
-
+#include <app_timer.h>
```

Comment:
Is this include (and the call to app_timer_init()) still needed? Maybe because the MotorController still use NRF timers?

Reply:
If I recall correctly, I had to add this include after I removed it from TimerController. App_timers are still used elsewhere, so the initialization is necessary. After this PR, we can take a look at #1095, which seems to remove the rest of the app_timers.